publish.html

```diff
@@ -670,24 +628,176 @@ <h2><a class="anchor" id="repository"></a>Using OMERO as a Public Repository</h2
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-<p>Full documentation on how to set up OMERO as a public repository and details of customisation options can be found in the <a href="https://docs.openmicroscopy.org/latest/omero/sysadmins/index.html#optimizing-omero-as-a-data-repository" target="_blank" class="external-url">OMERO Repository documentation</a>.</p>
+<p>
+    Full documentation on how to set up OMERO as a public repository and
+    details of customisation options can be found in the
+    <a href="https://docs.openmicroscopy.org/latest/omero/sysadmins/index.html#optimizing-omero-as-a-data-repository" target="_blank" class="external-url">OMERO Repository documentation</a>.
+    There is also documentation describing a
+    <a href="https://docs.openmicroscopy.org/latest/omero/sysadmins/publication.html"target="_blank" class="external-url">complete workflow for hosting data for a publication</a>.
```

Comment:
This is 404 just now, but I assume that will be in place.

Reply:
Yes, that's the new doc that is going out with 5.4.2